examples/ex_2_time_evolution.py

```diff
@@ -59,8 +63,11 @@ def main():
     with measure_time() as t_info:
         algorithm = qsp_time_evolution_algorithm(operator, evolution_time, 1e-3)
 
+    # Transpile the circuit to a Clifford+T circuit
+    # cliff_t_algorithm = algorithm.transpile_to_clifford_t()
+    cliff_t_algorithm = algorithm
```

Comment:
Is there a reason why line 67 is commented out? We should probably uncomment it or remove it.

Reply:
That's just a vestige of testing. I've uncommented it.